CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- Adds ability to use environment variables in Sensu Go.
```

Comment:
lets list out the env vars we enabled, this is not specific to `sensu-go` you can absolutely use ENV vars in the context of "sensu ruby".

Reply:
👍